en/docs/administer/control-egress-traffic-for-your-organization.md

```diff
@@ -0,0 +1,40 @@
+# Control Egress Traffic for your Organization
+
+In Choreo, you have the option to control egress traffic originating from your applications. You can control egress traffic at the organization level by defining an allow list or a deny list. Default egress behaviour is allow all which means egress traffic to any destination is allowed.
+
+## Configure egress rules at the organization level
```

Comment:
We might have to go with "Configure egress policies at the organization level" right?

Reply:
fixed with [Requested changes](https://github.com/wso2/docs-choreo-dev/pull/1637/commits/daed9e3ab828f115482aa08b02fba84af203895c)